tests/conftest.py

```diff
@@ -792,3 +696,28 @@ def example_sqlite_mixed_types_database(
     finally:
         # after completing the tests, remove the file
         pathlib.Path(filepath).unlink()
+
+
+@pytest.fixture(name="example_s3_path_csv_jump")
+def fixture_example_s3_path_csv_jump() -> str:
+    """
+    Provides an example s3 endpoint for use with tests
```

Comment:
It's a bit unclear to me why this isn't pointing to a csv file? Maybe you can clarify slightly in the docstring?

Reply:
Thanks good point! This points to a directory with CSV files within object storage. We point to the directory to help include all compartments and metadata CSV files for processing within CytoTable.